packages/iris-grid/src/IrisGrid.jsx

```diff
@@ -1024,6 +1024,18 @@ export class IrisGrid extends Component {
     }
   }
 
+  getAlwaysFetchColumns = memoize((alwaysFetchColumns, model) => {
+    const columnSet = new Set(alwaysFetchColumns);
+
+    model.columns.forEach(({ name }) => {
```

Comment:
This works, you could also build it just by looking at `model.floatingLeftColumnCount` and `model.floatingRightColumnCount` and then pulling from `model.columns`